### ISSUE TITLE
Dashboard filters

### ISSUE THREAD

**Issue by knopa** [HangfireIO/Hangfire#877]
Is there any way to find jobs with more that 1 attempts in dashboard?

**Comment by odinserj** [comment]
Nope, dashboard can show only jobs that are currently scheduled for yet another retry attempt.
